media-sound/mumble/mumble-9999.ebuild

```diff
@@ -64,94 +65,54 @@ BDEPEND="
 	virtual/pkgconfig
 "
 
-# NB: qmake does not support multilib but it's fine to configure
-# for the native ABI here
-src_configure() {
-	myuse() {
-		[[ -n "${1}" ]] || die "myuse: No use option given"
-		use ${1} || echo no-${1}
-	}
-
-	local conf_add=(
-		bundled-celt
-		no-bundled-opus
-		no-bundled-speex
-		no-embed-qt-translations
-		no-server
-		no-update
-		$(myuse alsa)
-		$(myuse dbus)
-		$(usex debug 'symbols debug' release)
-		$(myuse g15)
-		$(usex jack '' no-jackaudio)
-		$(myuse opus)
-		$(myuse oss)
-		$(myuse portaudio)
-		$(myuse pulseaudio)
-		$(myuse rnnoise)
-		$(usex speech '' no-speechd)
-		$(usex zeroconf '' no-bonjour)
-	)
-
-	use pch || conf_add+=( no-pch )
-
-	eqmake5 "${S}/main.pro" -recursive \
-		CONFIG+="${conf_add[*]}" \
-		DEFINES+="PLUGIN_PATH=/usr/$(get_libdir)/mumble"
+src_prepare() {
+        # required because of xdg.eclass also providing src_prepare
+        cmake_src_prepare
 }
 
-multilib_src_compile() {
-	local emake_args=(
-		# place libmumble* in a subdirectory
-		DESTDIR_ADD="/${MULTILIB_ABI_FLAG}"
-		{C,L}FLAGS_ADD="$(get_abi_CFLAGS)"
+src_configure() {
+
+	local mycmakeargs=(
+		"-Dalsa=$(usex alsa)"
+		"-DBUILD_TESTING=$(usex test)"
+		"-Dbundled-celt=ON"
+		"-Dbundled-opus=OFF"
+		"-Dbundled-speex=OFF"
+		"-Ddbus=$(usex dbus)"
+		"-Dg15=$(usex g15)"
+		"-Djackaudio=$(usex jack)"
+		"-Doverlay=ON"
+		"-Dportaudio=$(usex portaudio)"
+		"-Dpulseaudio=$(usex pulseaudio)"
+		"-Drnnoise=$(usex rnnoise)"
+		"-Dserver=OFF"
+		"-Dspeechd=$(usex speech)"
+		"-Dtranslations=$(usex nls)"
+		"-Dupdate=OFF"
+		"-Dzeroconf=$(usex zeroconf)"
 	)
-	# build only overlay library for other ABIs
-	multilib_is_native_abi || emake_args+=( -C overlay_gl )
-	emake "${emake_args[@]}"
-	emake clean
-}
 
-src_compile() {
-	multilib_foreach_abi multilib_src_compile
-}
-
-multilib_src_install() {
-	local dir=$(usex debug debug release)
-	insinto /usr/$(get_libdir)/${PN}
-	doins "${dir}/${MULTILIB_ABI_FLAG}"/libmumble.so*
-	if multilib_is_native_abi; then
-		dobin "${dir}"/mumble
-		doins "${dir}"/libcelt0.so* "${dir}"/plugins/lib*.so*
-	fi
+	cmake_src_configure
 }
 
 src_install() {
-	multilib_foreach_abi multilib_src_install
-
-	dodoc CHANGES README.md
-	dobin scripts/mumble-overlay
-
-	insinto /usr/share/services
-	doins scripts/mumble.protocol
-
-	domenu scripts/mumble.desktop
-
-	doicon -s scalable icons/mumble.svg
-
-	doman man/mumble-overlay.1
-	doman man/mumble.1
-}
-
-pkg_preinst() {
-	xdg_pkg_preinst
+	cmake_src_install
+
+	if use amd64 ; then
+		# The 32bit overlay library gets automatically built and installed on x86_64 platforms.
+		# Install it into the correct 32bit lib dir.
+		local libdir_64="/usr/$(get_libdir)/mumble"
+		local libdir_32="/usr/$(get_abi_var LIBDIR x86)/mumble"
+		mkdir -p ${D}/{$libdir_32} || die
```

Comment:
That line can be reduced by using dodir (sorry, didn't see it when writing my previous comment):
`dodir {$libdir_32}`
but no need to add this with another commit. I will change that when merging this PR.